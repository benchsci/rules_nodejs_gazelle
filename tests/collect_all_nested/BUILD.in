# gazelle:js_root
# gazelle:js_jest_config :jest.config
# gazelle:js_quiet
