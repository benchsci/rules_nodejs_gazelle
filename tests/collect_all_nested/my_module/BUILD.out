load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
load("@rules_jest//jest:defs.bzl", "jest_test")

# gazelle:js_collect_all
# gazelle:js_jest_shard_count 2
# gazelle:js_jest_size large

jest_test(
    name = "my_module_test",
    size = "large",
    srcs = [
        "nested_module/a.test.ts",
        "nested_module/b.test.ts",
    ],
    config = "//:jest.config",
    data = ["//:package_json"],
    shard_count = 2,
)

ts_project(
    name = "my_module",
    srcs = [
        "index.ts",
        "module_file.ts",
        "nested_module/index.ts",
        "nested_module/module_file.ts",
        "some_other_file.ts",
    ],
)
